symplyphysics/laws/electricity/circuits/resistivity_of_parallel_resistors.py

```diff
@@ -21,6 +25,57 @@
 law = Eq(parallel_resistance, 1 / SumArray(inv_resistances), evaluate=False)
 
 
+# Derive the law from the conductivity law for parallel resistors
+
+# Deriving the law for two resistors. Unfortunately, it is not possible to formally derive this for an arbitrary
+# number of resistors due to the proof limitation of sympy. But it is possible to follow the following technique
+# for any other number of resistors to prove the given equivalence.
+
+resistance1, resistance2 = tuple_of_symbols("resistance", units.impedance, 2)
+
+conductance1 = solve(
+    conductance_definition.definition.subs(
+        conductance_definition.object_resistance,
+        resistance1
+    ), 
+    conductance_definition.object_conductivity
+)[0]
+
+conductance2 = solve(
+    conductance_definition.definition.subs(
+        conductance_definition.object_resistance,
+        resistance2
+    ),
+    conductance_definition.object_conductivity
+)[0]
+
+parallel_conductance = solve(
+    parallel_conductivity.law.subs(
+        parallel_conductivity.conductances,
+        (conductance1, conductance2)
+    ),
+    parallel_conductivity.parallel_conductance
+)[0]
+
+parallel_resistance_from_conductivity_law = solve(
+    conductance_definition.definition.subs(
+        conductance_definition.object_conductivity,
+        parallel_conductance
+    ),
+    conductance_definition.object_resistance
+)[0]
+
+parallel_resistance_from_law_in_question = solve(
+    law.subs(inv_resistances, (1 / resistance1, 1 / resistance2)),
```

Comment:
We should not exlplictly divide, when proving something. Use conductance law instead.